catalog/acm/config-mgmt-iam.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   namespace: platform-namespace # kpt-set: ${platform-namespace}
   annotations:
     cnrm.cloud.google.com/blueprint: cnrm/acm/v0.0.1
-    cnrm.cloud.google.com/project-id: platform-project-id # kpt-set: ${platform-project-id}
+    cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
```

Comment:
Ideally this shouldn't be a setter at all—instead use set-project-id function.

Reply:
Yes, next version should have that all replaced. Since this is used in other blueprints that have the project-id setter still in place, it may be problematic to change this alone. Also our README template needs to be updated to include the the instruction on using the set-project-id function.